CHANGELOG.md

```diff
@@ -100,6 +100,7 @@ https://github.com/hyperledger/besu/releases/download/24.5.2/besu-24.5.2.zip / s
 - Add `rlp decode` subcommand to decode IBFT/QBFT extraData to validator list [#6895](https://github.com/hyperledger/besu/pull/6895)
 - Allow users to specify which plugins are registered [#6700](https://github.com/hyperledger/besu/pull/6700)
 - Layered txpool tuning for blob transactions [#6940](https://github.com/hyperledger/besu/pull/6940)
+- Enable continuous profiling with default setting [#7006](https://github.com/hyperledger/besu/pull/7006)
```

Comment:
need to move this to the upcoming release

Reply:
done [caff34f](https://github.com/hyperledger/besu/pull/7006/commits/caff34fde632cb34586078c3a2d5ef988f8383aa)